evoting/service/service.go

```diff
@@ -298,7 +299,7 @@ func (s *Service) GetElections(req *evoting.GetElections) (*evoting.GetElections
 
 // GetBox message handler to retrieve the casted ballot in an election.
 func (s *Service) GetBox(req *evoting.GetBox) (*evoting.GetBoxReply, error) {
-	election, err := lib.GetElection(s.skipchain, req.ID)
+	election, err := lib.GetElection(s.skipchain, req.ID, false, 0)
```

Comment:
A different way to have done this would have been to introduce a new API called lib.GetElectionCheckVoted. That way, GetElection calls would not end up with this useless 0 argument in them, which is a bit arbitrary/ugly/confusing.

I'm not recommending you change this now, just pointing out that it's not always bad to expand the public API: if the new function prevents callers of the old function from getting uglified, it's a nice tradeoff.